### PR TITLE
Add Quick External Video Share

### PR DESCRIPTION
**Pull request by cipi1965**
This PR implements the recognize of video links in slides (as with polls) and put it in a dropdown to quickly play it with external video player.

### CODE REVIEW

**Comment by antobinary** [comment]
Thank you for this contribution @cipi1965 !
Could you please confirm you have filled out a CLA? https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code ?

**Comment by cipi1965** [comment]
@antobinary yes, sent

File: bigbluebutton-html5/imports/ui/components/actions-bar/quick-videos-dropdown/component.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <Dropdown>
+      <DropdownTrigger tabIndex={0}>
+        <Button
+          aria-label="Video nella slide"
+          circle
+          className={styles.button}
+          color="primary"
+          hideLabel
+          icon="video"
+          label="Video nella slide"
```
Comment (fcecagno) [inline bigbluebutton-html5/imports/ui/components/actions-bar/quick-videos-dropdown/component.jsx:46]: Not a complete review, just a comment that this should be moved to the localization file in English, translations are done on Transifex.

Comment (cipi1965) [reply bigbluebutton-html5/imports/ui/components/actions-bar/quick-videos-dropdown/component.jsx:46]: You’re right, i forgot it

Comment (alialic) [reply bigbluebutton-html5/imports/ui/components/actions-bar/quick-videos-dropdown/component.jsx:46]: hi,I want to customize of the meeting page. But I can not access the session codes. Do you know the access path of the codes related to the session page?

**Comment by ramonlsouza** [comment]
closing (similar to #15946, now merged in 2.6)

**Pull request closed without merging**